GentWebApi/Models/Coupon.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace GentWebApi.Models {
+	public class Coupon {
+
+		public string Id { get; set; }
+		public string ImageUrl { get; set; }
+
```

Comment:
Heeft een coupon geen promotionId of Promotion-object?

Reply:
daar twijfel ik nog over, want je moet een kortingsbon ook aan een user kunnen geven ofzo. dus het is nie per se gelinkt aan een promotie denk ik

Reply:
aha is juist hmm